rust/main/agents/scraper/migration/src/m20230309_000004_create_table_gas_payment.rs

```diff
@@ -90,6 +87,19 @@ impl MigrationTrait for Migration {
             )
             .await?;
 
+        manager
+            .create_index(
+                Index::create()
+                    .table(GasPayment::Table)
+                    .name("gas_payment_origin_interchain_gas_paymaster_sequence_idx")
+                    .col(GasPayment::Origin)
+                    .col(GasPayment::InterchainGasPaymaster)
+                    .col(GasPayment::Sequence)
```

Comment:
if the message has no `Sequence` would the index still be effective? Maybe there's some other column we could use if not

Reply:
I will use the index to find gas payments by sequence for sequence-aware indexing. If a record does not have a sequence, it means that watermark-aware indexing is used and we don't need to use this index.